src/UITests/README.md

```diff
@@ -0,0 +1,30 @@
+## UI Tests
+* UI tests reside in the `UITests` project.
```

Comment:
Can we also add brief information about possible artifacts in ADO?
- a11ytest file
- events file
- screen recording

Reply:
Info added